common/perception_utils/include/perception_utils/object_classification.hpp

```diff
@@ -0,0 +1,41 @@
+
+// Copyright 2022 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef PERCEPTION_UTILS__OBJECT_CLASSIFICATION_HPP_
+#define PERCEPTION_UTILS__OBJECT_CLASSIFICATION_HPP_
+
+#include "autoware_auto_perception_msgs/msg/object_classification.hpp"
+
+namespace perception_utils
+{
+using autoware_auto_perception_msgs::msg::ObjectClassification;
+
+bool isVehicle(const uint8_t object_classification)
```

Comment:
I guess if you use `vehicle` then the bicycle is also treated as a vehicle.
I'd recommend asking native speakers for naming.

```suggestion
bool isCar(const uint8_t object_classification)
```

Reply:
Thank you for thoughtful review.

I check the definition of vehicle like <https://www.nuscenes.org/nuscenes#data-annotation>.
So I fix isVehicle() function in [e6c22e2](https://github.com/autowarefoundation/autoware.universe/pull/1908/commits/e6c22e26d6986815c678839378f76bb3270b99c2).

In addition, I add new function isCarLikeVehicle() function in [2087bea](https://github.com/autowarefoundation/autoware.universe/pull/1908/commits/2087beae8a140127e9c2bf797468dc21b7d728c0),  consulting with @yukke42 .